README.md

```diff
@@ -104,6 +104,34 @@ you arguments into migrate/migrate running in a container.
 - Migrate the task-svc database one down: `./migrate.sh task-svc down 1`
 - Current migration version of the task-svc database: `./migrate.sh task-svc version`
 
+## Enviroment variables
+Every service should contain a `.env.example` file which should be copied to a usable `.env` file
+
+Default Variables:
```

Comment:
Should we keep the general overview of common environment variables?

Reply:
I would say so having more information available should be better than fewer, but if we notice it's too confusing we can always remove it. So I would keep it for now